packages/flame_behaviors/pubspec.yaml

```diff
@@ -2,6 +2,9 @@ name: flame_behaviors
 description: Flame Behaviors applies separation of concerns to game logic in the form of Entities and Behaviors, built by Very Good Ventures.
 version: 0.1.1
 homepage: https://github.com/VeryGoodOpenSource/flame_behaviors
+repository: https://github.com/VeryGoodOpenSource/flame_behaviors
+issue_tracker: https://github.com/VeryGoodOpenSource/flame_behaviors/issues
+documentation: https://github.com/VeryGoodOpenSource/flame_behaviors/tree/main/docs
```

Comment:
💯

Reply:
I did uncomment the documentation one so that the pipeline can succeed, but will be fixed in the next PR